src/components/cards/providers/Subreddit.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FaReddit } from 'react-icons/fa';
+
+import { Card, Repeater, DedicatedAvailability } from '../core';
+
+const SubredditCard: React.FC<{ query: string }> = ({ query }) => {
+  const { t } = useTranslation();
+  const normalizedQuery = normalize(query, {
```

Comment:
import `normalize`
Also always run `tsc --noEmit` before committing to make sure there are no other errors.

Reply:
Wow, I'm really tired :sweat_smile: